tests/test_linux.py

```diff
@@ -274,3 +274,33 @@ def test_serialize_state_with_closed_files(self):
         fd = platform.sys_open(filename, os.O_RDONLY, 0o600)
         platform.sys_close(fd)
         pickle.dumps(platform)
+
+    def test_thumb_mode_entrypoint(self):
+        # thumb_mode_entrypoint is a binary with only one instruction
+        #   0x1000: add.w   r0, r1, r2
+        # which is a Thumb instruction, so the entrypoint is set to 0x1001
+        m = Manticore.linux(os.path.join(os.path.dirname(__file__), 'binaries', 'thumb_mode_entrypoint'))
+        m.success = True
```

Comment:
Wouldn't it be better to initialize this with `False` too? So if hooks doesn't fire for some reason, we will fail the test.

Reply:
Done in my latest commit.